utils/build/docker/nodejs/express4/iast/index.js

```diff
@@ -120,6 +120,19 @@ function init (app, tracer) {
     })
   });
 
+
+  app.get('/iast/insecure-cookie/test_insecure', (req, res) => {
```

Comment:
```suggestion
  app.get('/iast/insecure-cookie/test_secure', (req, res) => {
```

Reply:
You're right, but the change is in the other endpoint. Changed :-)